create_derivatives/routines.py

```diff
@@ -297,7 +297,6 @@ def create_manifest(self, identifier, obj_data):
         manifest.set_metadata({"Date": obj_data["dates"]})
         manifest.thumbnail = self.set_thumbnail(self.jp2_files[0].stem)
         sequence = manifest.sequence(ident=identifier)
-        annotation_count = 1
         for jp2_file in self.jp2_files:
             page_number = get_page_number(jp2_file)
```

Comment:
It looks like we're not using the padded page number here anymore so you can just `lstrip` once here and not have to do it twice below.